doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,108 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. Each
+entity is represented by a JSON object which fields are the following ones:
+
+* `id` and `type` are the entity ID and entity type, respectively. In the case the entity has an attribute named
+  `id` and/or `type` then they are ignored. For entities without type (i.e. type equals to "" internally), the
+  `type` field is not included
+* All other first level fields correspond to the names of the attributes of the entity.
+  * If the attribute type allows "compact form" and the attribute doesn't have any metadata element, the value of the
+    attribute field is the attribute value itself.
+  * If the attribute type doesn't allow "compact form" and/or the attribute have at least a metadata element, the value
+    of the attrigue field is JSON object including the following fields: `value` (always), `type` (except if the
+    type is empty) and other fields, each one corresponding to the name of one of the metadatas of the attribute.
+* Metadata fields follows the same representation rules that applies to attributes fields (except that a metadata
+  cannot have metadata elements associated).
+
+Canonical format description (TBD).
+
+### Special attribute types
+
+Types are case-insensitive, e.g. `number` is considered an special type, but `Number` is considered a user-defined
+type.
+
+* `number`:  identifies attribute meaning numbers, either integer or float. They are associated to JSON number type.
+They can be used in greater than, lesser than, greater or equal, lesser or equal and range filters. They support compact form:
+
+```
+"temperature": 32.5
+```
+
+* `string`, which represents text string. They are associated to JSON string type. They support compact form:
+
+```
+"msg": "Hello world"
+```
+
+* `boolean`, which represent a logical value (either true or false). They are associated to JSON boolean type.
+They support compact form:
+
+```
+"active": true
+```
+
+* `date`:  identifies attribute meaning a date, in ISO8601 format. They can be used in greater than,
+lesser than, greater or equal, lesser or equal and range filters.
+
+
+* `geo:point`: identifies an attribute meaning a geolocation. They are associated to JSON string type. An
```

Comment:
The statement about "an entity can only have one geo:point attribute" is to avoid that an entity may be in two places at the same time. That could break the way in which geolocation works. What do you think?